### DISCUSSION TITLE
i18n in https://www.alci.dev/

### DISCUSSION THREAD

**Discussion by alcibiadesc** [sveltekit-i18n/lib#178] [Show and tell]
Hi! 

I wanted to share with you that we I implemented internationalization (i18n) on my website [alci.dev](https://www.alci.dev/). Now, visitors can enjoy content in different languages according to their preferences. 

if you're looking to check the implementation just take a look and use the language selector in the top right corner to change the language to your liking.

Thanks to the sveltekit-i18n library <3

**Comment by jarda-svoboda** [comment]
Hi @alcibiadesc! Thanks for sharing this. I have great news for you: v3 is here!
	•	Built on Svelte 5 runes — no more stores, $t('key') is now i18n.t('key')
	•	One instance per request on the server, so a visitor can't get another one's language
	•	Pluggable parser — Curly Message Format by default, ICU if you prefer
	•	Typed keys and payloads through config.schema
	•	invalidate() and per-locale cache for translations that change under you
	•	All examples rebuilt — locale-router-advanced is the closest to what your site does
Migrating from v2 covers the move.
Closing this discussion as outdated — feel free to let me know how your experience with v3 goes! Cheers :)
